Assets/Scripts/MRCCameraUpdate.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright 2021 The Open Brush Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using UnityEngine;
+
+using Node = UnityEngine.XR.XRNode;
+using NodeState = UnityEngine.XR.XRNodeState;
+
+#if OCULUS_SUPPORTED
+
+public class MRCCameraUpdate : MonoBehaviour {
+
+    private OVRPose? calibratedCameraPose = null;
+
+    void Update() {
+        if (!calibratedCameraPose.HasValue) {
+            if (!OVRPlugin.Media.GetInitialized())
+                return;
+
+            OVRPlugin.CameraIntrinsics cameraIntrinsics;
+            OVRPlugin.CameraExtrinsics cameraExtrinsics;
+
+            if (OVRPlugin.GetMixedRealityCameraInfo(0, out cameraExtrinsics, out cameraIntrinsics)) {
+                calibratedCameraPose = cameraExtrinsics.RelativePose.ToOVRPose();
+            } else {
+                return;
+            }
+        }
+
+        OVRPose cameraStagePoseInUnits = calibratedCameraPose.Value;
+
+        // Converting position from meters to decimeters (unit used by Open Brush)
+        cameraStagePoseInUnits.position *= TiltBrush.App.METERS_TO_UNITS;
+
+        // Workaround to fix the OVRExtensions.ToWorldSpacePose() and OVRComposition.ComputeCameraWorldSpacePose()
+        // when computing the Mixed Reality foreground and background camera positions.
+        OVRPose headPose = OVRPose.identity;
+
+        Vector3 pos;
+        Quaternion rot;
+        if (OVRNodeStateProperties.GetNodeStatePropertyVector3(Node.Head, NodeStatePropertyType.Position, OVRPlugin.Node.Head, OVRPlugin.Step.Render, out pos))
```

Comment:
Nit: If we are to stay in line with Tilt Brush's code style, this line and line 54 should be split to stay within 100 chars long.

Reply:
Thank you @TimAidley. I've just updated the code style for this class to keep it consistent with the project, please let me know if this is ok.

Reply:
Looks good!